tests/integration/suite/test_pod_security_policies.py

```diff
@@ -43,6 +43,25 @@ def create_pspt(client):
     return pspt
 
 
+def set_psp_enabled(k8s_dynamic_client, value):
```

Comment:
Including the `wait_for(lambda: api_client.by_id_cluster(id="local").capabilities.pspEnabled)` in here would clean up the other code since it's always done anyways.